src/js/modules/formReset/test/index.test.js

```diff
@@ -0,0 +1,149 @@
+import $ from '@justeat/f-dom';
+import TestUtils from 'js-test-buddy';
+import { init, resetInputs } from '../index';
+
+describe('module init', () => {
+    it('is a function', () => {
+        expect(typeof init).toBe('function');
+    });
+
+    it('will not error if no [data-js-reset] is available', () => {
+        // Arrange
+        TestUtils.setBodyHtml('');
+
+        // Act
+        init();
+        const button = $('[data-js-reset]');
+
+        // Assert
+        expect(button.length).toBe(0);
+    });
+
+    it('will uncheck a radio buttons from from button trigger bound on init()', () => {
+        // Arrange
+        TestUtils.setBodyHtml(`
+            <a data-js-reset="testGroup"></a>
+            <input type="radio" data-js-reset-input="testGroup" checked/>
+        `);
+
+        // Act
+        init();
+        const [button] = document.querySelectorAll('[data-js-reset]');
+        TestUtils.click(button);
+
+        // Assert
+        const [radio] = document.querySelectorAll('[data-js-reset-input]');
+        expect(radio.checked).toBe(false);
+    });
+
+    it('will clear multiple form elements from button trigger bound on init()', () => {
```

Comment:
Could you put this in a describe block then for each expect below scope it to one single `IT`.


```
describe('When multiple form elements are cleared', () => {
    it('should uncheck a radio form element from button trigger bound on init()', () => {
        TestUtils.setBodyHtml(`
                    <a data-js-reset="testGroup"></a>
                    <input type="radio" data-js-reset-input="testGroup" checked/>
                `);
        const [radio] = document.querySelectorAll('[data-js-reset-input]');
        expect(radio.checked).toBe(false);
    })

    ...
    ...
});
```